wmn-data.json

```diff
@@ -4987,6 +4987,17 @@
         "cat" : "XXXPORNXXX",
         "valid" : true
        },
+       {
+        "name" : "Replit",
+        "uri_check" : "https://replit.com/@{account}",
+        "e_code" : 200,
+        "e_string" : "Copy profile link",
+        "m_code" : 404,
+        "m_string" : "404 Not Found",

```

Comment:
I'm not seeing this string in the response. Perhaps use `<title>Replit - 404 - Replit</title>` instead? Although, that only is returned via a web browser request. A `curl` request for an incorrect user has a response of `{"message":"User not found","name":"NotFoundError","status":404}`

Reply:
Thanks for raising it, I trusted the check script to strongly. 
So I had the same response from CURL vs browser, I'll modify it to the browser version.